ui/src/tool-panel/km-post/geometry-km-post-linking-infobox.tsx

```diff
@@ -76,7 +76,8 @@ const GeometryKmPostLinkingInfobox: React.FC<GeometryKmPostLinkingInfoboxProps>
                       40,
                   )
                 : [];
-        }, [geometryKmPost.trackNumberId, geometryKmPost.location, layoutKmPost]) || [];
+        }, [publishType, geometryKmPost.trackNumberId, geometryKmPost.location, layoutKmPost]) ||
```

Comment:
Jäin jotenkin miettimään tätä. Käytännössähän tällä haulla haetaan kilometripylväiden linkitysboksin km-pylväät, ka. listaus näytetään vain linkitystilassa, ja linkitystilaan meno laittaa aina DRAFT-moden päälle -> sinällään tähän voisi vaan kovakoodata DRAFT-haun ja unohtaa tuon publishType-parametrin kokonaan. Toisaalta tässä nykytavassa on se hyvä puoli, että kun haku riippuu publishTypestä, niin se suoritetaan aina kun mennään luonnostilaan. Tällöin kilometripylväslistauksella on mahis olla tuoreempi jos operaattori on seissyt ei-luonnostilassa pitkään ennen linkitystä. Km-pylväiden changeTimeen sitomalla tuo kanssa hoituisi, mutta toisaalta en välttämättä haluaisi että listaus latautuu uudestaan kesken linkityksen, joten en välttämättä sitoisi lopulta siihenkään... 🤔 

Aikamoista fiinistelyä tuo mun mietintä, mutta jos joku muu keksii tuohon fiksun ratkaisun niin sanokaa vaan. Mutta hyvä (ja ennen kaikkea aikaisempaa toteutusta parempi) tää on tosiaan näinkin, joten ei tarvi tähän liikaa takertua 👍